frontend/src/views/camp/NavigationCamp.vue

```diff
@@ -1,100 +1,216 @@
+        <span class="sr-only-sm-and-down">{{
+          $tc("views.camp.navigationCamp.print")
+        }}</span>
       </v-btn>
       <v-btn :to="campRoute(camp(), 'admin')" text>
         <v-icon :left="$vuetify.breakpoint.mdAndUp">mdi-cogs</v-icon>
-        <span class="sr-only-sm-and-down">{{ $tc('views.camp.navigationCamp.admin') }}</span>
+        <span class="sr-only-sm-and-down">{{
+          $tc("views.camp.navigationCamp.admin")
+        }}</span>
       </v-btn>
     </v-toolbar-items>
     <v-spacer />
     <user-meta />
   </v-app-bar>
-  <v-bottom-navigation v-else grow
-                       app background-color="blue-grey darken-4"
-                       dark>
-    <v-btn :to="campRoute(camp(), 'program')">
-      <span>{{ $tc('views.camp.navigationCamp.program') }}</span>
-      <v-icon>mdi-view-dashboard</v-icon>
-    </v-btn>
-    <v-btn :to="campRoute(camp(), 'story')">
-      <span>{{ $tc('views.camp.navigationCamp.story') }}</span>
-      <v-icon>mdi-book-open-variant</v-icon>
-    </v-btn>
-    <v-btn :to="campRoute(camp(), 'dashboard')">
-      <span>{{ camp().name }}</span>
-      <v-icon large>mdi-tent</v-icon>
-    </v-btn>
-    <v-btn :to="campRoute(camp(), 'material')">
-      <span>{{ $tc('views.camp.navigationCamp.material') }}</span>
-      <v-icon>mdi-package-variant</v-icon>
-    </v-btn>
-    <v-btn :to="campRoute(camp(), 'admin')">
-      <span>{{ $tc('views.camp.navigationCamp.admin') }}</span>
-      <v-icon>mdi-account-cog</v-icon>
-    </v-btn>
-  </v-bottom-navigation>
+  <div v-else>
+    <v-bottom-navigation grow app
+                         background-color="blue-grey darken-4"
+                         dark>
+      <v-btn :to="campRoute(camp(), 'program')">
+        <span>{{ $tc("views.camp.navigationCamp.program") }}</span>
+        <v-icon>mdi-view-dashboard</v-icon>
+      </v-btn>
+      <v-btn :to="campRoute(camp(), 'story')">
+        <span>{{ $tc("views.camp.navigationCamp.story") }}</span>
+        <v-icon>mdi-book-open-variant</v-icon>
+      </v-btn>
+      <v-btn :to="campRoute(camp(), 'dashboard')">
+        <span>{{ camp().name }}</span>
+        <v-icon large>mdi-tent</v-icon>
+      </v-btn>
+      <v-btn :to="campRoute(camp(), 'material')">
+        <span>{{ $tc("views.camp.navigationCamp.material") }}</span>
+        <v-icon>mdi-package-variant</v-icon>
+      </v-btn>
+      <v-btn @click="open = true">
+        <span>{{ $tc("views.camp.navigationCamp.more") }}</span>
+        <v-icon>mdi-menu</v-icon>
+      </v-btn>
+    </v-bottom-navigation>
+    <v-navigation-drawer v-model="open" app
+                         right
+                         width="300">
+      <div class="d-flex flex-column fill-height">
+        <h2
+          class="text-h6 text-center d-flex flex-column align-center blue-grey darken-4 white--text py-6">
+          <v-icon x-large>$vuetify.icons.ecamp</v-icon>
+          <span> eCamp </span>
```

Comment:
I would argue to use a translation here as well. I would just never write plain displayed text into the code.

Reply:
As this is our product name, I don't want to translate this. Also in the logo component it's not translated as well.